docs/user-guide/kubernetes.md

```diff
@@ -510,6 +510,13 @@ You should now be able to visit the websites in your browser.
 * [cheeses.minikube/cheddar](http://cheeses.minikube/cheddar/)
 * [cheeses.minikube/wensleydale](http://cheeses.minikube/wensleydale/)
 
+## Forwarding to ExternalNames
+
+When specifying an [ExternalName](https://kubernetes.io/docs/concepts/services-networking/service/#services-without-selectors),
+Træfik will forward requests to the given host accordingly and use HTTPS when the Service port matches 443. This still requires setting up a proper port mapping on the Service from the Ingress port to the (external) Service port.
+
+NB: In the future, the target Service port will likely be used to determine the protocol and specify the port to forward requests to.
```

Comment:
WIth @errm on this one. Especially since many new users are looking at this guide.